plugins/UserCountryMap/Controller.php

```diff
@@ -265,8 +266,7 @@ private function getMetrics($idSite, $period, $date, $token_auth)
         $metrics = array();
         if (!empty($metaData[0]['metrics']) && is_array($metaData[0]['metrics'])) {
             foreach ($metaData[0]['metrics'] as $id => $val) {
-                // todo: should use SettingsPiwik::isUniqueVisitorsEnabled ?
-                if (Common::getRequestVar('period') == 'day' || $id != 'nb_uniq_visitors') {
+                if (SettingsPiwik::isUniqueVisitorsEnabled(Common::getRequestVar('period')) || $id != 'nb_uniq_visitors') {
```

Comment:
In case you work on this anyway it could be nice to make the code more readable and assign like meaningful variables. like `$period = Common...`. also for `$id != 'nb_uniq_visitors'`